package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nmp",
-  "version": "1.0.0",
+  "version": "3.0.0",
```

Comment:
no need to bupm a version without making `RELEASE_NOTES.md` or following a release workflow:
https://www.npmjs.com/package/standard-version

Reply:
Thanks, will be considered in the future. On prod project, we provide release notes for each release